cmd/loop/liquidity.go

```diff
@@ -0,0 +1,174 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/lightninglabs/loop/looprpc"
+	"github.com/urfave/cli"
+)
+
+var getLiquidityParamsCommand = cli.Command{
+	Name:  "getparams",
+	Usage: "show liquidity manager parameters",
+	Description: "Displays the current set of parameters that are set " +
+		"for the liquidity manager.",
+	Action: getParams,
+}
+
+func getParams(ctx *cli.Context) error {
+	client, cleanup, err := getClient(ctx)
+	if err != nil {
+		return err
+	}
+	defer cleanup()
+
+	cfg, err := client.GetLiquidityParams(
+		context.Background(), &looprpc.GetLiquidityParamsRequest{},
+	)
+	if err != nil {
+		return err
+	}
+
+	printRespJSON(cfg)
+
+	return nil
+}
+
+var setLiquidityParamCommand = cli.Command{
+	Name:        "setparam",
+	Usage:       "set liquidity manager rule for a channel",
+	Description: "Update or remove the liquidity rule for a channel.",
+	ArgsUsage:   "shortchanid",
+	Flags: []cli.Flag{
+		cli.IntFlag{
+			Name: "mininbound",
+			Usage: "the minimum percentage of inbound liquidity " +
+				"to total capacity beneath which to " +
+				"recommend loop out to acquire inbound.",
+		},
+		cli.IntFlag{
+			Name: "minoutbound",
+			Usage: "the minimum percentage of outbound liquidity " +
+				"to total capacity beneath which to " +
+				"recommend loop in to acquire outbound.",
+		},
+		cli.BoolFlag{
+			Name:  "clear",
+			Usage: "remove the rule currently set for the channel.",
+		},
+	},
+	Action: setParam,
+}
+
+func setParam(ctx *cli.Context) error {
+	// We require that a channel ID is set for this rule update.
+	if ctx.NArg() != 1 {
+		return fmt.Errorf("please set a channel id for the rule " +
+			"update")
+	}
+
+	chanID, err := strconv.ParseInt(ctx.Args().First(), 10, 64)
+	if err != nil {
+		return fmt.Errorf("could not parse channel ID: %v", err)
+	}
+
+	client, cleanup, err := getClient(ctx)
+	if err != nil {
+		return err
+	}
+	defer cleanup()
+
+	// We need to set all values in the config that we params the server to
```

Comment:
nit: that first sentence got scrambled somehow, not sure what you want to say here :)